tdrs-frontend/package.json

```diff
@@ -105,8 +109,10 @@
     "collectCoverageFrom": [
       "src/**/*.{js,jsx}",
       "!src/**/index.js",
+      "!src/utils/createFileInputErrorState.js",
       "!src/serviceWorker.js",
       "!src/configureStore.js",
+      "!src/mirage.js",
```

Comment:
Ignoring this because this file merely creates the test mirage server and is not relevant.